clap_derive/Cargo.toml

```diff
@@ -33,6 +33,8 @@ syn = { version = "2.0.8", features = ["full"] }
 quote = "1.0.9"
 proc-macro2 = "1.0.69"
 heck = "0.5.0"
+pulldown-cmark = { version = "0.12.2", default-features = false }
+anstyle = "1.0.10"
```

Comment:
A way to merge this more quickly is to have an `unstable-markdown` feature enabling this

Reply:
yes, this would definitely be a good idea